packages/ui/src/helpers/auth.ts

```diff
@@ -144,3 +144,15 @@ export const getSendEventAliases = (send: Sender<AuthEvent>) => {
     submit: sendToMachine('SUBMIT'),
   } as const;
 };
+
+export const getServiceFacade = ([state, send]) => {
+  const { user } = state.context as AuthContext;
```

Comment:
`state.context.user` isn't always up to date as `actorState.context.user` because the parent state context isn't updated until the actor finishes (this can be improved). 

This is okay with the use cases in this PR, but can be a problem later if customer modifies other screens like setup-totp.

```suggestion
  const { user } = getActorContext(state) as AuthActorContext;
```

Reply:
A long term solution would be have a single source of truth that contains all up-to-date context in both parent and child actors -- basically 
```tsx
const getContext(state) = {
  const parentContext = state;
  const childContext = getActorContext(state);
  return { ...parentContext, ...childContext }; // child context should take precedence.
}
```
Added this to e2 tasks.

Reply:
That sounds desirable, actually?  What I'm hearing is that even though the user is logged in, the Authenticator is still going through its Setup TOTP flow, so the `user` isn't yet available because the machine isn't _done_.

That sounds like customers could do something like this then:

```js
function App() {
  const { user } = useAuthenticator();

  if (!user) {
    return <Authenticator>;
  }

  return <RestOfMyApp />
}
```

Reply:
Yeah, I see your point, especially for `{user, signOut}` use case. Let's wait until we actually see a need to do so!